src/worker.rs

```diff
@@ -68,7 +68,7 @@ pub struct AppComputeWorker<W: ComputeWorker> {
     render_queue: RenderQueue,
     cached_pipeline_ids: HashMap<String, CachedAppComputePipelineId>,
     pipelines: HashMap<String, Option<ComputePipeline>>,
-    buffers: HashMap<String, Buffer>,
+    pub buffers: HashMap<String, Buffer>,

```

Comment:
@tombh I would prefer (if possible) to keep this private. Are there specific methods we could attach to this to the builder to allow for the types of buffers you would want to add?

Reply:
That was easy, I just added a `get_buffer` method.